src/pages-and-resources/teams/Settings.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FieldArray } from 'formik';
+import { v4 as uuid } from 'uuid';
+import * as Yup from 'yup';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Button, Form, Icon } from '@edx/paragon';
+import { Add } from '@edx/paragon/icons';
+import { TeamSetTypes, TeamSizes } from '../../data/constants';
+
+import FormikErrorFeedback from '../../generic/FormikErrorFeedback';
+import { setupYupExtensions, useAppSetting } from '../../utils';
+import AppSettingsModal from '../app-settings-modal/AppSettingsModal';
+import messages from './messages';
+import TeamSetEditor from './TeamSetEditor';
+
+setupYupExtensions();
+
+function TeamSettings({
+  intl,
+  onClose,
+}) {
+  const [teamsConfiguration, saveSettings] = useAppSetting('teamsConfiguration');
+  const blankNewTeamSet = {
+    name: '',
+    description: '',
+    type: TeamSetTypes.OPEN,
+    maxTeamSize: TeamSizes.DEFAULT,
+    id: null,
+  };
+
+  const handleSettingsSave = async (values) => {
+    // For newly-added teams, fill in an id.
+    const teamSets = values.topics?.map(teamSet => ({
+      id: teamSet.id || uuid(),
+      name: teamSet.name,
+      type: teamSet.type,
+      description: teamSet.description,
+      max_team_size: teamSet.maxTeamSize,
+    }));
+    return saveSettings({ topics: teamSets, max_team_size: values.maxTeamSize });
```

Comment:
@mehaknasir  This is here the setting are being saved. I'm checking again to see in case the recent changes broke something.

Reply:
I've tested it, and it seems to be working.

Reply:
Thank you for the update. I worked with another teammate he is able to save settings too. on patch call on enabling teams however we got this error. I think you might be aware of that have a look.

<img width="782" alt="Screen Shot 2021-09-15 at 2 15 14 PM" src="https://user-images.githubusercontent.com/67791278/133407543-5f8f86b5-c2be-4012-b104-af1f69bd6c1a.png">
<img width="1377" alt="Screen Shot 2021-09-15 at 2 15 33 PM" src="https://user-images.githubusercontent.com/67791278/133407556-7b667705-8d58-48af-bae9-a3fc296a5f1e.png">

Reply:
@mehaknasir  That API needs to be updated now that the MFE is implemented. Will create a ticket for that. 
However, simply configuring teams should enable it.